packages/gatsby/src/gatsby-node.js

```diff
@@ -0,0 +1,20 @@
+exports.onCreateWebpackConfig = ({ plugins, actions }) => {
+  actions.setWebpackConfig({
+    plugins: [
+      plugins.define({
+        __SENTRY_RELEASE__: JSON.stringify(
+          // GitHub Actions - https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables
+          process.env.GITHUB_SHA ||
+            // Netlify - https://docs.netlify.com/configure-builds/environment-variables/#build-metadata
+            process.env.COMMIT_REF ||
+            // Vercel - https://vercel.com/docs/v2/build-step#system-environment-variables
+            process.env.VERCEL_GITHUB_COMMIT_SHA ||
```

Comment:
@dcramer Does it make sense to check other Vercel source code provider environment variables like `VERCEL_BITBUCKET_COMMIT_SHA`?

Reply:
Ah yeah good catch

Reply:
Vercel also supports Gitlab btw